README.md

```diff
@@ -12,8 +15,15 @@ Docs are still a work in progress, but you can view the [API docs](https://githu
 - [Installation](#installation)
 - [Builds](#builds)
 - [Examples](#examples)
+  + [Initializing the Client](#initializing-the-client)
+  + [Fetching Products](#fetching-products)
+  + [Fetching Collections](#fetching-collections)
+  + [Creating, Adding, Updating, and Removing Items from a Checkout](#creating-adding-updating-and-removing-items-from-a-checkout)
```

Comment:
It should be creating a checkout and adding line items to a checkout. I'll reword the sections to make that clearer. Thanks for the catch! 😄